templates/terraform/examples/tpu_node_full.tf.erb

```diff
@@ -33,3 +33,22 @@ resource "google_tpu_node" "<%= ctx[:primary_resource_id] %>" {
     preemptible = true
   }
 }
+
+data "google_compute_network" "network" {
+  name = "default"
+}
+
+resource "google_compute_global_address" "service_range" {
+  provider = google-beta
```

Comment:
Does this need to use `beta` here? I think we want all the providers in this config to be the same.

Reply:
fixed